public/js/reader.js

```diff
@@ -109,7 +111,38 @@ Reader.loadImages = function () {
             if (Reader.infiniteScroll) {
                 Reader.initInfiniteScrollView();
             } else {
-                $(document).on("click.imagemap_change_pages", "#Map area", Reader.handlePaginator);
+                $(window).on('wheel', function(e) {
+                    if(Reader.isFullscreen) {
```

Comment:
This doesn't seem to work properly in manga/reverse mode.

Reply:
i need check that